client/state/ui/notices/actions.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+	REMOVE_NOTICE
+} from 'state/action-types';
+
+import { uniqueId } from 'lodash';
+
+export function createNoticeAction( status, text, options = {} ) {
+	return {
+		noticeId: uniqueId(),
+		duration: options.duration,
+		showDismiss: ( typeof options.showDismiss === 'boolean' ? options.showDismiss : true ),
+		type: NEW_NOTICE,
+		status: status,
+		text: text
+	};
+}
+
+export function removeNoticeAction( noticeId ) {
+	return {
+		noticeId: noticeId,
+		type: REMOVE_NOTICE
+	};
+}
+
+export function noticesMapDispatchToProps( dispatch ) {
```

Comment:
`mapDispatchToProps` functions feel out of place in `actions.js`, since they're very much tied to React in that they define a component's props bound to action creators. In this case, if the desire is to have a named function, I think it's reasonable to define in the same file as the component, directly above the call to `connect`.

```jsx
const NoticesList = React.createClass( { /* ... */ } );

function mapStateToProps( state ) {
	// ...
}

function mapDispatchToProps( dispatch ) {
	// ...
}

export default connect( mapStateToProps, mapDispatchToProps )( NoticesList );
```

While I could see some re-use between these mapping functions, I don't think it's a convention we should promote, as it might lead us to creating very generic mapping connectors that pass complex handlers intended to handle a variety of use-cases. Instead, I think `connect` arguments should be specialized to the particular component its wrapping.